website/docs/plugin/testing/acceptance-tests/plan-checks.mdx

```diff
@@ -21,13 +21,15 @@ A **plan check** is a test assertion that inspects the plan file at a specific p
 
 The `terraform-plugin-testing` module provides a package [`plancheck`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/plancheck) with built-in plan checks for common use-cases:
 
-| Check                                                                                                                                                   | Description                                                                  |
-|---------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------|
-| [`plancheck.ExpectEmptyPlan()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/plancheck#ExpectEmptyPlan)                             | Asserts the entire plan has no operations for apply.                         |
-| [`plancheck.ExpectNonEmptyPlan()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/plancheck#ExpectNonEmptyPlan)                       | Asserts the entire plan contains at least one operation for apply.           |
-| [`plancheck.ExpectResourceAction(address, operation)`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/plancheck#ExpectResourceAction) | Asserts the given resource has the specified operation for apply.            |
-| [`plancheck.ExpectUnknownValue(address, path)`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/plancheck#ExpectUnknownValue)          | Asserts the specified attribute at the given resource has an unknown value.  |
-| [`plancheck.ExpectSensitiveValue(address, path)`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/plancheck#ExpectSensitiveValue)      | Asserts the specified attribute at the given resource has a sensitive value. |
+| Check                                                                                                                                                      | Description                                                                  |
```

Comment:
It might be good to split up this table (and maybe the page) to be resource attribute checks versus output value checks to reduce potential confusion.

Reply:
Have split the page into Plan Checks - Overview, Resource, Output, and Custom.

The docs for `ExpectEmptyPlan` and `ExpectNonEmptyPlan` currently appear on the Resource page. Perhaps these should be on a separate page or on the Overview page?

Reply:
Also ❤️ for thinking ahead to just split up the pages.

Reply:
Have added a `General Plan Checks` section to the Overview page to include plan checks for `ExpectEmptyPlan`/`ExpectNonEmptyPlan`.